src/main/java/tc/oc/pgm/rotation/Rotation.java

```diff
@@ -0,0 +1,130 @@
+package tc.oc.pgm.rotation;
+
+import com.google.common.base.Optional;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.map.PGMMap;
+
+public class Rotation implements PGMMapOrderProvider {
+  private ConfigurationSection configurationSection;
+
+  private String name;
+  private boolean enabled;
+  private List<PGMMap> maps = new ArrayList<>();
+  private int players;
+  private int position;
+
+  Rotation(ConfigurationSection configurationSection, String name) {
+    this.configurationSection = configurationSection;
+    this.name = name;
+    this.enabled = configurationSection.getBoolean("enabled");
+    this.players = configurationSection.getInt("players");
+    this.position = configurationSection.getInt("position");
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
```

Comment:
Are the `setName` `setMaps` and `setPlayers` methods necessary? When would this ever be used?

Reply:
Well the `setName` & `setMaps` I will remove, makes no sense to keep them there. But I was planning on creating a command for changing rotation's player count trigger from in-game

Reply:
Removed. https://github.com/Electroid/PGM/pull/114/commits/99cce8ad7f45acaedc80accac8a3329cf30dca1d